azurelinuxagent/common/protocol/wire.py

```diff
@@ -1277,6 +1334,80 @@ def parse(self, xml_text):
         return self
 
 
+class RemoteAccess(object):
+    """
+    Object containing information about user accounts
+    """
+    #
+    # <RemoteAccess>
+    #   <Version/>
+    #   <Incarnation/>
+    #    <Users>
+    #       <User>
+    #         <Name/>
+    #         <Password/>
+    #         <Expiration/>
+    #         <Groups>
+    #           <GroupName/>
+    #           ...
+    #         </Groups>
+    #       </User>
+    #     </Users>
+    #   </RemoteAccess>
+    #
+
+    def __init__(self, xml_text):
+        logger.verbose("Load RemoteAccess.xml")
+        self.Users = []
+        self.Incarnation = None
+        if xml_text is not None:
+            self.parse(xml_text)
+
+    def parse(self, xml_text):
```

Comment:
Please use the same pattern as the existing code, see GoalState for an example.

Reply:
done